docs/docs/product-documentation.md

```diff
@@ -75,53 +71,48 @@ The strategic aim of the Swapchain team is to develop an HTLC-backed cryptocurre
 
 #### 2.2.2. User scenario
 
-As a user I am either in possession of BTC, BTS or both. Let us assume I am in possession of BTCs. However, I would like to exchange BTC for BTS so that I am also in possession of BTS. Since I want to avoid counterparty risks and I do not trust people I do not know, I want to have maximum guarantees that my cryptocurrency exchange meets these requirements. Swapchain offers my desired services and satisfies my needs. Hence, I create a user account where minimal personal information is needed. Once registered, I can log in and check the orderbook for potential swap offers. Moreover, I can add my own swap offer to the orderbook. If I find a fitting swap offer or if someone accepts my previously posted swap offer, I start the ACCS by confirming the order in the orderbook. Now, the ACCS is triggered. If everything goes well, I have successfully exchanged BTC for BTS. If my swap partner tries to fool the system or the time lock runs out of time, the transaction is cancelled and I receive my refunded BTC.
-
-#### 2.2.3. UX style guide
-
-A draft of the UI mockup can be found with the following link: https://framer.com/share/swapchain-8fTM7MfGO6B5Axijk3Dy.
+As a user I am either in possession of BTC, BTS or both. Let us assume I am in possession of BTCs. However, I would like to exchange BTC for BTS so that I am also in possession of BTS. Since I want to avoid counterparty risks and I do not trust people I do not know, I want to have maximum guarantees that my cryptocurrency exchange meets these requirements. As I already have an exchange partner, Swapchain offers my desired services and satisfies my needs. I simply access the Swapchain app and put my information, such as currency and amount I want to swap, transaction fee priority to influence the speed of my block being mined and my private and public key to sign the transaction. Subsequently, the app generates a secret hash that serves as the hashlock on the created ACCS. I pass this hashed secret to my exchange partner so he/she can put the same hash lock on the counterpart transaction. Now, the ACCS is triggered. If everything goes well, I have successfully exchanged BTC for BTS. If my swap partner tries to fool the system or the time lock runs out of time, the transaction is cancelled and I receive my refunded BTC.
```

Comment:
"Subsequently, the app generates ..." --> Technically we do not have a web app. I would rather say platform.

Reply:
"Since I want to avoid counterparty risks and I do not trust people I do not know"
->
Since I do not want to trust the counterparty,...

Reply:
"transaction fee priority to influence the speed of my block being mined and my private and public key to sign the transaction"
->
"priority to increase or decrease the probability that my transaction gets included in the next block and my private key to sign the transaction"
(there's no public key needed for signing)

Reply:
> "Subsequently, the app generates ..." --> Technically we do not have a web app. I would rather say platform.

Oh we do indeed have a web app. "Subsequently, the app generates a secret hash" is perfect for me.